src/components/pages/pseudochoice.tsx

```diff
@@ -12,7 +52,44 @@ function PseudoChoice(): JSX.Element {
         <div id="level-title">Level Title</div>
         <div id="instructions-title">Instructions</div>
         <div id="instructions">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</div>
-        <div id="content">INSERT SIDEBAR CONTENT HERE</div>
+        <div id="content">INSERT SIDEBAR CONTENT HERE
+          <AllChoices
+            number1 = "3"
```

Comment:
Can we think of a more efficient way to pass this information? Maybe a 2D array or 2 arrays of numbers and booleans.